sdf/1.9/segmentation_camera.sdf

```diff
@@ -0,0 +1,18 @@
+<element name="segmentation" required="0">
+  <description>These elements are specific to an segmentation camera sensor.</description>
+
+  <element name="segmentation_type" type="string" default="semantic" required="0">
+    <description>
+      The segmentation type of the segmentation camera, the valid options:
+        - semantic: Semantic segmentation which provides a gray image which its pixels
+                    have the label value of the objects, and a colored image which its pixels
+                    have a unique color for each label.
+
+        - panoptic | instance: Panoptic segmentation which provides an image where each pixel
+                               has 1 channel for label value of the object and 2 channels for the
+                               number of the instances of that label, and a colored image which its
+                               pixels have a unique color for each instance.
```

Comment:
```suggestion
        - panoptic | instance: Panoptic segmentation, which provides 2 images:
                               1. A 3-channel image, where each pixel has 1 channel for the label value of the object and 2 channels for the number of the instances of that label
                               2. A colored image, with the pixel values being a unique color for each instance
```

@chapulina or @azeey: for the 3-channel image described here, should we specify which channel is associated with the label and which 2 channels are associated with the instance count? Or should we not define this here so that it can be defined by downstream users of SDF?